build-bin/docker-compose-zipkin-eureka.yml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2015-2024 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+# uses 2.4 so we can use condition: service_healthy
+version: "2.4"
+
+# Test both authenticated and unauthenticated, as if there is a Spring problem,
+# the latter will crash. We only need to use HEALTHCHECK for this.
+services:
+  eureka:
```

Comment:
without a compose, it effectively only runs this part. I daisy-chained the two paths since we are using spring security which is wonky. Maybe someone can convert this to armeria and use normal simple basic auth filter one day, but I went with the spring-security route initially.

Reply:
Sorry missed that

Reply:
no worries I should have checked. Anyway the prior one failed on health check for frustrating reasons.